irayalina/static/scss/_variable.scss

```
@@ -0,0 +1,13 @@
+$color-primary: rgba(44,83,131,1);
+$color-secondary:rgba(95,175,201,1);
```

Ity tokony `rgba(214,40,40,1);`
```suggestion
$color-secondary:rgba(214,40,40,1);
```